sqladmin/models.py

```diff
@@ -283,6 +288,88 @@ class UserAdmin(ModelAdmin, model=User):
     edit_template: ClassVar[str] = "edit.html"
     """Edit view template. Default is `edit.html`."""
 
+    # Form
+    form: ClassVar[Optional[Type[Form]]] = None
```

Comment:
I think technically you're right to have Optional here, but since we're not using it as arguments it won't be checked? What do you think?

Reply:
So basically, all the other Optionals I think are fine to remove. But I don't think this one should be removed.

Reply:
Yes, you're right.